volume/ceph/ceph.go

```diff
@@ -250,7 +258,8 @@ func (v *Volume) Unmount(id string) error {
 	}
 
 	if fsType == cryptoLuksFsType {
-		cmd := exec.Command("cryptsetup", "luksClose", v.Name())
+		luksDevMapperName := getLuksDeviceMapperName(v.Name())
+		cmd := exec.Command("cryptsetup", "luksClose", luksDevMapperName)
 		if err := cmd.Run(); err != nil {
 			unmapCephVolume(v.name, v.mappedDevicePath)
```

Comment:
Thoughts on adding
``` defer unmapCephVolume(v.name, v.mappedDevicePath) ``` 
at the top of the Unmount? I feel like no matter what we should unmap ceph. Then at least for debug reasons we will be able to see the error in the flapping job